Core/Src/analyzer.c

```diff
@@ -182,16 +178,10 @@ nertimer_t ocvTimer;
 
 bool is_first_reading_ = true;
 
-/* private function prototypes */
-void disable_therms();
-void high_curr_therm_check();
-void diff_curr_therm_check();
-void calc_state_of_charge();
-
 /* we are not corrctly mapping each therm reading to the correct cell. So, we are taking the average of all good readings (not disabled) for a given chip, 
  and assigning that to be the cell val for every cell in the chip*/
 
-void calc_cell_temps()
+void calc_cell_temps(acc_data_t *bmsdata)
```

Comment:
ok, can ignroe comment about not having a local bmsdata object if we are passing all in. im okay with this

Reply:
This will be changed eventually with proper data passing between threads.